src/main/java/org/openrewrite/kotlin/tree/K.java

```diff
@@ -1436,16 +1438,49 @@ public List<TypeParameter> getTypeParameters() {
         @Nullable
         TypeConstraints typeConstraints;
 
-        @Nullable
-        J.MethodDeclaration getter;
+        // A replacement of `getter`,`setter` and `isSetterFirst`
+        JContainer<J.MethodDeclaration> accessors;
 
         @Nullable
-        J.MethodDeclaration setter;
+        JRightPadded<Expression> receiver;
 
-        boolean isSetterFirst;
+        // For backward compatibility, handle removed fields `getter`, 'setter' and `isSetterFirst` which has been relocated to `accessors`
+        // Todo, Remove when all kotlin LSTs have been rebuilt.
+        @JsonCreator
+        public Property(@JsonProperty("id") UUID id,
+                        @JsonProperty("prefix") Space prefix,
+                        @JsonProperty("markers") Markers markers,
+                        @JsonProperty("typeParameters") @Nullable JContainer<TypeParameter> typeParameters,
+                        @JsonProperty("variableDeclarations") VariableDeclarations variableDeclarations,
+                        @JsonProperty("typeConstraints") @Nullable K.TypeConstraints typeConstraints,
+                        @JsonProperty("getter") @Nullable J.MethodDeclaration getter,
+                        @JsonProperty("setter") @Nullable J.MethodDeclaration setter,
+                        @JsonProperty("isSetterFirst") boolean isSetterFirst,
+                        @JsonProperty("receiver") @Nullable JRightPadded<Expression> receiver
```

Comment:
Also here we should test with a SaaS-like configuration, which shouldn't require these `@JsonProperty` annotations.

Reply:
Verified on SAAS and confirmed that `@JsonProperty` is optional. so I removed the redundant `@JsonProperty` for  other fields but kept 3 removed fields to have explicit `@JsonProperty` (it's also optional but I prefer to keep it to emphasize they are removed fields, anyway this method will be removed later after new LSTs are built)